data/core/src/commonMain/kotlin/com/tunjid/heron/data/network/models/TimelineItemConversions.kt

```diff
@@ -67,6 +69,24 @@ internal fun FeedViewPost.feedItemEntity(
         null,
         -> post.indexedAt
     },
+    embeddedRecordId = when (val embed = post.embed) {
```

Comment:
This was a typo from me 😞. Should be `embeddedRecordUri`, not `embededRecordId`. There'sno need to create a new `RecordId` type.